src/models/searchservice/response/FeaturedSnippetDirectAnswer.ts

```diff
@@ -17,6 +17,8 @@ export interface FeaturedSnippetDirectAnswer extends DirectAnswer {
   relatedResult: Result;
```

Comment:
Do we need to mark the `value` above as optional? Or does that come from the `inheritDoc`?